app/views/answers/_answer.html.erb

```diff
@@ -0,0 +1,27 @@
+<p>
+  <%= form_for answer, url: answer_votes_path(answer), method: :post do |f| %>
```

Comment:
If you use form_for here, you could use form_for with `answer.votes.new` (or a variable that represents that) and rails would infer the right URL and method. The other thing you could do is use `button_to(answer_votes_path(answer, vote: 'up')` and rails would generate the form for you.